### DISCUSSION TITLE
I digg the datepicker

### DISCUSSION THREAD

**Discussion by codewithmenow** [mdbootstrap/TW-Elements#1408] [Kind words & general feedback]
Love your datepicker component:
https://tailwind-elements.com/docs/standard/forms/datepicker/

The native one is very difficult to style, and that's exactly the look I was looking for.

I hope you will add a daterange option some day, so that we'll be able to choose, a "start" and "end" dates within a single input.

**Comment by filipkappa** [comment]
Thanks for your feedback!

Make sure to add your idea to our Ideas discussion category: #1390

**Comment by jdnichollsc** [comment]
@filipkappa wondering, why the Cancel button is not restoring the previous selection? what do you think about having this behavior?

**Reply by filipkappa** [reply]
It's fixed! Take a look at the datepicker now & let us know if you still experience it.
https://tailwind-elements.com/docs/standard/forms/datepicker/

Also - make sure to create a support discussion next time:
https://github.com/mdbootstrap/Tailwind-Elements/discussions/categories/support-from-community

**Reply by jdnichollsc** [reply]
@filipkappa thanks for your amazing support, looks awesome mate! ❤️
